doc/SCC-API-(Implemented).md

```diff
@@ -1177,6 +1177,15 @@ Status: 200 OK
 - `free` ( *Boolean* ): true for free products, false for ones that require their own subscription
 
 
+##### Errors:
+
+- 422: `The requested product '%s' is not activated on this system.`
+- 422: `Required parameters are missing or empty: installed_products`
+- 500: `No base product found` - When the `installed_products` parameter contains no base product.
+- 500: `Multiple base products found: ["SUSE Linux Enterprise Server 12 SP4 x86_64", "SUSE Linux Enterprise Server 12 SP3 x86_64"]` -
+When the `installed_products` parameter contains more than one base product.
+This should not happen, since a system can only have one base product installed.
+
 ##### List System Offline Migrations
 
 Given a list of installed products, return all possible online migration paths.
```

Comment:
It should say "return all possible offline migration paths." here, right?
Can you maybe fix that in this PR, @lagartoflojo

Reply:
Nice catch, thanks!